packages/@aws-cdk/aws-codepipeline/lib/action.ts

```diff
@@ -119,7 +119,79 @@ export interface ActionConfig {
 }
 
 /**
- * A Pipeline Action
+ * A Pipeline Action.
```

Comment:
Can you move it to the ReadMe file? I think this is too complicated for the JSDocs of an interface...

Reply:
Sure thing :)

Reply:
@skinny85 check now.